atomicapp/utils.py

```diff
@@ -197,6 +197,36 @@ def sanitizePath(path):
             return path[7:]
 
     @staticmethod
+    def run_cmd(cmd, args_str=None):
+        """
+        Runs a command with its arguments.
+
+        Args:
+            args_str (str): arguments to the command.
```

Comment:
This is weird because args_str this is actually being passed as `stdin` to the command and not actually as args. Since nothing is actually using it I would remove it.